proposals/dao/fip_33.ts

```diff
@@ -0,0 +1,217 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '../../types/types';
+import { TransactionResponse } from '@ethersproject/providers';
+
+before(async () => {
+    chai.use(CBN(ethers.BigNumber));
+});
+
+// Constants
+// CR oracle wrapper
+const CR_KEEPER_INCENTIVE = ethers.constants.WeiPerEther.mul(1000); // 1000 FEI
+
+// CR oracle guardian
+const CR_GUARDIAN_FREQUENCY = 12 * 60 * 60 // 12 hours
+const CR_GUARDIAN_DEVIATION_BPS = '500' // 5%
+
+// LBP swapper
+const LBP_FREQUENCY = '604800'; // weekly
+const MIN_LBP_SIZE = ethers.constants.WeiPerEther.mul(100_000); // 100k FEI
+
+// PCV Equity Minter
+const PCV_EQUITY_MINTER_INCENTIVE = ethers.constants.WeiPerEther.mul(1000); // 1000 FEI
+const PCV_EQUITY_MINTER_FREQUENCY = '604800'; // weekly
+const PCV_EQUITY_MINTER_APR_BPS = '1000'; // 10%
+
+/*
+
+TRIBE Buybacks
+
+DEPLOY ACTIONS:
+
+1. Deploy CR Oracle Keeper
+2. Deploy TRIBE/ETH Oracle
+3. Deploy TRIBE/USD Oracle
+4. Deploy TRIBE LBP Swapper
+5. Create TRIBE LBP pool
+6. Init TRIBE LBP Swapper
+7. Deploy PCV Equity Minter
+8. Deploy CR Oracle Guardian
+
+DAO ACTIONS:
+1. Make PCV Equity Minter a minter
+2. Make CR Oracle Keeper a minter
+3. Seed LBP Swapper with TRIBE
+4. Create ORACLE_ADMIN role
+5. Set ORACLE_ADMIN role to admin for CR Oracle
+6. Set ORACLE_ADMIN role to admin for CR Oracle Wrapper
+7. Grant Oracle Admin role to Collateralization Oracle Guardian
+8. TODO ORACLE admin to OA ??
```

Comment:
Need to decide if we want OA to have admin powers on the oracles. It kind of makes sense because they may add new FEI deployments that need to be added

Reply:
@Joeysantoro what can oracle-admins affect?

Reply:
Seems okay for now. We may want to discuss this more as a team since this is a lot of power.